examples/standalone_single_project/main.tf

```diff
@@ -0,0 +1,263 @@
+  cloudbuild_cd_repo        = "${var.app_name}-cloudbuild-cd-config"
+  deploy_branch_clusters    = local.deploy_branch_clusters
+  app_deploy_trigger_yaml   = "cloudbuild-cd.yaml"
+  cache_bucket_name         = module.ci_pipeline.cache_bucket_name
+  cloudbuild_private_pool   = module.cloudbuild_private_pool.workerpool_id
+  clouddeploy_pipeline_name = local.clouddeploy_pipeline_name
+  depends_on = [
+    module.ci_pipeline
+  ]
+}
+
+# Cloud Build Private Pool
+module "cloudbuild_private_pool" {
+  source = "../../modules/cloudbuild-private-pool"
+
+  project_id                = local.project_id
+  network_project_id        = local.project_id
+  location                  = var.region
+  create_cloudbuild_network = true
+  private_pool_vpc_name     = "cloudbuild-worker-vpc"
+  worker_pool_name          = "cloudbuild-workerpool"
+  machine_type              = "e2-highcpu-32"
+
+  worker_address    = "10.39.0.0"
+  worker_range_name = "cloudbuild-worker-range"
+}
+
+# Cloud Build Workerpool <-> GKE HA VPNs
+module "gke_cloudbuild_vpn" {
+  for_each = local.gke_net_vpn
+
+  source = "../../modules/workerpool-gke-ha-vpn"
+
+  project_id = local.project_id
+  location   = var.region
+
+  gke_project             = each.value.project_id
+  gke_network             = each.value.network
+  gke_location            = each.value.location
+  gke_control_plane_cidrs = each.value.control_plane_cidrs
+
+  workerpool_network = module.cloudbuild_private_pool.workerpool_network
+  workerpool_range   = module.cloudbuild_private_pool.workerpool_range
+  gateway_1_asn      = each.value.gateway_1_asn
+  gateway_2_asn      = each.value.gateway_2_asn
+  bgp_range_1        = each.value.bgp_range_1
+  bgp_range_2        = each.value.bgp_range_2
+}
+
+###### Private Clusters ######
+# Private Cluster VPCs
+module "vpc" {
+  for_each = toset(local.envs)
+  source   = "terraform-google-modules/network/google"
+  version  = "~> 4.0"
+
+  project_id   = local.project_id
+  network_name = "${var.app_name}-vpc-${each.value}"
+  routing_mode = "REGIONAL"
+
+  subnets = [
+    {
+      subnet_name           = "${var.app_name}-subnet-${each.value}"
+      subnet_ip             = "10.${local.ip_increment[each.value]}.0.0/17"
+      subnet_region         = var.region
+      subnet_private_access = "true"
+    },
+  ]
+  secondary_ranges = {
+    "${var.app_name}-subnet-${each.value}" = [
+      {
+        range_name    = "${var.region}-01-gke-01-pods"
+        ip_cidr_range = "192.168.0.0/18"
+      },
+      {
+        range_name    = "${var.region}-01-gke-01-services"
+        ip_cidr_range = "192.168.64.0/18"
+      },
+    ]
+  }
+}
+
+resource "google_compute_network_peering_routes_config" "gke_peering_routes_config" {
+  for_each = toset(local.envs)
+
+  project = local.project_id
+  peering = module.gke_cluster[each.value].peering_name
+  network = module.vpc[each.value].network_name
+
+  import_custom_routes = true
+  export_custom_routes = true
+}
+
+module "gke_cluster" {
```

Comment:
nit: since this a long main.tf, lets split it up as gke.tf, network.tf and cicd.tf with respective locals in those files.